### PR TITLE
:rotating_light: Build integration test

### PR DESCRIPTION
**Pull request by AndreMiras**
Also fixes CHANGELOG.md descending PR order

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -36,20 +37,24 @@
 
 **Merged pull requests:**
 
-- Android gradle build: fix apk name [\#1025](https://github.com/kivy/buildozer/pull/1025) ([SomberNight](https://github.com/SomberNight))
```
Comment (AndreMiras) [inline CHANGELOG.md:39]: Jus reordered from ascending to descending

File: .github/workflows/test_python.yml [hunk 2]
```diff
@@ -62,6 +62,10 @@ jobs:
       run: |
         sed "s/# android.accept_sdk_license = False/android.accept_sdk_license = True/" -i buildozer.spec
         buildozer android p4a -- --help
+    - name: buildozer android debug
+      run: |
+        touch main.py
+        buildozer android debug
```
Comment (AndreMiras) [inline .github/workflows/test_python.yml:68]: Adds a 11 minutes overhead, but it's probably worth trying until we have proper unit test coverage :cry:

Comment (opacam) [reply .github/workflows/test_python.yml:68]: Oh boy, this is quite of time, but you are right, better this than nothing :+1:

**Comment by AndreMiras** [comment]
Yes indeed it's quite some time and most of what we do here is already tested in p4a. But for the lack of having nice unit test coverage we integration test. That will give us confidence that at least the minimum is working.
Thank for the review! :rocket:

**Pull request merged**